app/assets/javascripts/darkswarm/cookies_banner/cookies_banner.html.haml

```diff
@@ -0,0 +1,15 @@
+.row
+  .large-9.columns
+    %p
+      {{ 'legal.cookies_banner.cookies_usage' | t}}
```

Comment:
These translations are actually under `legal.cookies_policy`. It would be better to use the lazy lookup and move the translations to the right place.

Reply:
These translations are not in legal.cookies_policy, they are in legal.cookies_banner, which is different. Or I may be missing something.
These are javascript translations in angular templates where I don't think we can use lazy lookup. Or I may be missing something... (if this is correct, we can't use lazy lookup in angular templates, I can add this detail to the i18n page).

Reply:
You are right. I just checked and this is correct. And since this is an Angular template, it needs to be translated through the `t` filter. Somehow my computer is not re-compiling the locale for Javascript and I get the missing translations. But that's not your fault. ;-)

Reply:
👍